internal/verifier/recheck.go

```diff
@@ -91,6 +77,11 @@ func (verifier *Verifier) insertRecheckDocs(
 		verifier.logger.Debug().Msgf("Persisted %d recheck doc(s) for generation %d", len(models), generation)
 	}
 
+	// Silence any duplicate key errors as recheck docs should have existed.
+	if mongo.IsDuplicateKeyError(err) {
```

Comment:
Dupe key errors can exist alongside other errors.

We should probably instead look for non-simple dupe-key errors so that anything that _isn’t_ a dupe-key will still be handled.